js/components/authoring.js

```diff
@@ -54,14 +55,36 @@ export default class Authoring extends PureComponent {
     }
   }
 
+  handleBackButtonClick () {
+    const { step } = this.state
+    if (step === 2) {
+      this.unloadModel()
+    } else if (step === 3) {
+      this.setStep2()
+    }
+  }
+
   handleInteractionChange (interactionName) {
     const { setOption } = this.props
     setOption('interaction', interactionName)
   }
 
   loadModel (name) {
-    const { loadModel, setOption } = this.props
+    const { loadModel } = this.props
     loadModel(name)
+    this.setStep2()
+  }
+
+  unloadModel () {
+    const { unloadModel, setOption } = this.props
+    unloadModel()
+    setOption('interaction', 'none')
+    setOption('selectableInteractions', [])
+    this.setState({step: 1})
+  }
+
+  setStep2 () {
```

Comment:
I refactored the common code into a single method here! It wasn't as straightforward to pull out a method for setting step 1, since most options are set from initialization the first time it's loaded. So, I left `unloadModel` alone.